error_reporting/main_test.py

```diff
@@ -17,7 +17,9 @@
 import main
 
 
-@mock.patch("fluent.event")
-def test_error_sends(event_mock):
+@mock.patch("main.error_reporting")
+def test_error_sends(error_reporting_mock):
+    client_mock = mock.Mock()
```

Comment:
with the mock we actually test *nothing* here. The client library can change and break the sample and tests would never fail, which is why we don't write unit tests here. Everything in this repo is a system test so I do not care about the cost of making an RPC.

If you want to assert that the call is being made, use [`wraps`](http://wesmckinney.com/blog/spying-with-python-mocks/).